schema/image/normalize.js

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
Yeah, I dunno; i like the way the resulting code reads, but this big import statement is ugly. Though we do this for GraphQL too...